client/src/Views/ProfilePage.jsx

```diff
@@ -1,20 +1,99 @@
+// ? Add Instant Username Check
 import React, {Component} from "react";
+import axios from "axios";
 
 class ProfilePage extends Component {
   state = {
-    test: null,
+    isLoading: true,
+    userId: this.props.userId,
+    inputs: {
+      username: null,
+      password: null,
+      email: null,
+      profileImage: null,
+    },
   };
+
+  componentDidMount() {
+    const userId = localStorage.getItem("userId");
+    const data = {
+      headers: {
+        Authorization: localStorage.getItem("token"),
```

Comment:
Could get tedious to keep adding this. Maybe use `axios.defaults.headers.common['Authorization'] = [token];` instead.